src/main/webapp/app/lecture/lecture.component.ts

```diff
@@ -111,6 +120,17 @@ export class LectureComponent implements OnInit {
         this.applyFilters();
     }
 
+    sortRows() {
+        //Sort alphabetically for title and description
+        if (this.predicate == 'title')
```

Comment:
Why do we need this? I'm pretty sure we can assume the tile to be always existent, isn't it?

Reply:
Yes, you are right. This check is unnecessary and not needed.